spec/myst/string_spec.mt

```diff
@@ -145,3 +145,98 @@ describe("String#to_s") do
     assert("\tone\ttwo".to_s == "\tone\ttwo")
   end
 end
+
+describe("String#chars") do
+  it("Returns a List with chars in a string") do
+    assert("abc".chars == ["a", "b", "c"])
+    assert("yay".chars == ["y", "a", "y"])
+  end
+end
+
+describe("String#downcase") do
+  it("Returns a lowercased version of itself") do
+    assert("HELLO".downcase == "hello")
+  end
+end
+
+describe("String#upcase") do
+  it("Returns a uppercased version of itself") do
+    assert("hello".upcase == "HELLO")
+  end
+end
+
+describe("String#chomp") do
+  it("Returns self with a newline at the end removed when no argument is provided") do
+    assert("Bob likes to fish\n".chomp == "Bob likes to fish")
+  end
+
+  it("Returns self with specified string removed from the end") do
+    assert("Fish does not like to be fish".chomp(" fish") == "Fish does not like to be")
+  end
+end
+
+describe("String#strip") do
+  it("Returns a new string with all leading and trailing whitespace removed") do
+    assert("   Hello \t".strip == "Hello")
+    assert((("  \t   \n    " * 7) + "Find me if you can" + ("\n" * 28)).strip == "Find me if you can")
+  end
+end
+
+describe("String#lstrip") do
+  it("Returns a new string with all leading whitespace removed") do
+    assert("   Hello \t".lstrip == "Hello \t")
+    assert("\t\tBye tabs".lstrip == "Bye tabs")
+  end
+end
+
+describe("String#rstrip") do
+  it("Returns a new string with all trailing whitespace removed") do
+    assert("   Hello \t".rstrip == "   Hello")
+    assert(" Nah\n\n\n".rstrip == " Nah")
+  end
+end
+
+describe("String#includes") do
+  it("Returns true if the string contains given string") do
+    assert("Bob fish? yeah".includes?("fish?") == true)
+    assert("Chess is awesome".includes?("Poker") == false)
+  end
```

Comment:
Same here. One test for a truthy assertion, another for a falsey assertion.

Also, for boolean assertions, you don't need the `== true`. `assert` will only pass if the result is the literal `true` boolean. There's also `refute` for asserting that something is `false`.

Reply:
Thanks for all the reviews! :D